packages/twenty-front/src/modules/action-menu/actions/record-actions/components/RegisterRecordActionEffect.tsx

```diff
@@ -17,7 +17,7 @@ export const RegisterRecordActionEffect = ({
   action,
   objectMetadataItem,
 }: RegisterRecordActionEffectProps) => {
-  const { shouldBeRegistered, onClick, ConfirmationModal } = action.actionHook({
+  const { shouldBeRegistered, onClick, ConfirmationModal } = action.useAction({
```

Comment:
For your information, the ESLint rule is only triggered when the hook is accessed like this:

![CleanShot 2025-01-29 at 17 12 46@2x](https://github.com/user-attachments/assets/24d64a06-bbaa-4ae5-ba3c-1a7d8319d4c7)

I think it's not required to make a change, but it would help readability to use the hook as any usual hook. We already used this pattern here, for instance: https://github.com/twentyhq/twenty/blob/5412a97af71d48b5fa0407b0537b2772b1636267/packages/twenty-front/src/modules/activities/emails/right-drawer/components/RightDrawerEmailThread.tsx#L64-L67